source/core/coverage/proftab.sql

```diff
@@ -1,4 +1,9 @@
-create table plsql_profiler_runs
+declare
```

Comment:
We use a pattern of one script for one object so this script is the only one breaking the pattern